config/cmake/HDF5PluginCache.cmake

```diff
@@ -8,7 +8,7 @@
 set (H5PL_BUILD_TESTING ON CACHE BOOL "Enable H5PL testing" FORCE)
 set (BUILD_EXAMPLES ON CACHE BOOL "Build H5PL Examples" FORCE)
 
-set (HDF5_HDF5_HEADER "H5pubconf.h" CACHE STRING "Name of HDF5 header" FORCE)
+set (H5PL_HDF5_HEADER "H5pubconf.h" CACHE STRING "Name of HDF5 header" FORCE)
 set (HDF5_LINK_LIBS ${HDF5_LIBSH_TARGET} CACHE STRING "HDF5 target" FORCE)
```

Comment:
I think this one also needs to be updated if I understand correctly.

Reply:
Maybe, I need to see where that is used but most likely it needs to be prefixed with "H5PL_"